arroyo/processing/processor.py

```diff
@@ -94,7 +96,12 @@ def on_partitions_revoked(partitions: Sequence[Partition]) -> None:
                 # assigned and is not closed or errored
                 try:
                     if self.__consumer.tell().keys() - set(partitions):
-                        _create_strategy()
+                        active_partitions = {
+                            partition: offset
+                            for partition, offset in self.__consumer.tell().items()
```

Comment:
This won't actually happen or cause issues today, but I think it's better to call `consumer.tell()` only once in this function and then use the result in both of these 2 places to avoid the risk of the value ever changing in between.